manifests/repo.pp

```diff
@@ -21,7 +21,7 @@
     case $facts['os']['family'] {
       'RedHat': {
         $majorrelease = $facts['os']['release']['major']
-        if (versioncmp(fact('os.release.major'), '7') >= 0 and $zabbix_version == '7.0') {
+        if versioncmp($zabbix_version, '7.0') >= 0 {
```

Comment:
the whole block still doesn't correct to me. first we check on the zabbix version, then on the OS version and then an else block?